aten/src/ATen/native/UnaryOps.h

```diff
@@ -6,6 +6,7 @@
 
 namespace at {
 class Tensor;
```

Comment:
Nit. I believe this is not needed anymore.

Reply:
It's used by `multinomial_with_replacement_stub`.